### ISSUE TITLE
completed-docs should enable @inheritdoc by default

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [palantir/tslint#3356]
As per the comment by @eric-parsons in https://github.com/palantir/tslint/issues/1921#issuecomment-307460035. 

> While it would be nice to have the ability to fine tune which tags are whitelisted as suggested above, I would think that "@inheritdoc" in particular should automatically and always bypass this rule, due to the inherent meaning of that tag. Without this, this rule is pretty much unusable for me right now, which is a shame since I really need it.

**Comment by ajafff** [comment]
That may not be necessary, because TypeScript is about to change the behavior when `@inheritdoc` is present: https://github.com/Microsoft/TypeScript/pull/18804
According to the discussion there, I might be possible that documentation is always inherited, even without the tag.

**Comment by JoshuaKGoldberg** [comment]
Interesting and exciting. I would think there should still be handling in TSLint for TS <=2.6 here, and the behavior of the rule should inherit whatever TypeScript adopts.

**Comment by jan-auer** [comment]
Any update on this? The linked PR in Typescript was merged in Nov 2017.

**Comment by JoshuaKGoldberg** [comment]
@jan-auer blocked on https://github.com/palantir/tslint/pull/3557 so far - but I still plan on adding it!

**Comment by FlorinAsavoaie** [comment]
I think the blocker was fixed...

**Comment by JoshuaKGoldberg** [comment]
@FlorinAsavoaie thanks for the bump! I personally don't plan on adding this, but accepting PRs if it can come in before November 1st (#4534).

**Comment by JoshuaKGoldberg** [comment]
☠️ *TSLint's time has come!* ☠️ 

TSLint is no longer accepting most feature requests per #4534. See **[typescript-eslint.io](https://typescript-eslint.io)** for the new, shiny way to lint your TypeScript code with **ESLint**. ✨ 

It was a pleasure open sourcing with you all!
